src/UIHandler.h

```diff
@@ -0,0 +1,131 @@
+#pragma once
+#ifndef UIHANDLER_H
+#define UIHANDLER_H
+
+#include "debug.h"
+
+#include "cursesdef.h"
+#include "output.h"
+
+#include "enums.h"
+#include <memory>
+
+enum class Sizes {
+    Minimum,
+    Prefered
+};
+
+class UIUtils
+{
+    public:
+        static void DrawBorder( WINDOW *wf_win, point offset, point m_thisSize );
+        static void DrawTab( WINDOW *wf_win, point offset, int tabOffset, bool tabActive,
+                             std::string text );
+};
+
+class UIPanel
```

Comment:
Out of curiosity (and also requested guidance from @kevingranade): why this class, function and member naming style. Much of the code uses all lowercase with underscores for class and function names.

Also: maybe the classes should all go into a `ui` namespace instead of having the `UI` in their name.